### DISCUSSION TITLE
Why is next server comsuming 25% of my memory?

### DISCUSSION THREAD

**Discussion by quackquavk** [vercel/next.js#70404] [Help]
### Summary

Why is next server comsuming 25% of my memory. My macbook m2 pro was literally lagging. Does this mean I should stop using Next JS altogether? For anyone curious I was working on this: https://chessdream.vercel.app . Its pretty heavy duty with all the chess board and chess engine running but its shouldn't cause my work machine that doesn't lag while im 3D rendering alongside video editing. 
<img width="938" alt="Screenshot 2024-09-24 at 20 23 58" src="https://github.com/user-attachments/assets/66b62636-2695-4c73-9e1b-e32875c6f516">

**Comment by icyJoseph** [comment]
Maybe you have a memory leak? Hard to tell without a repository to look at though.
